index.html

```diff
@@ -0,0 +1,220 @@
+                    <h3>Concert</h3>
+                    <p>Mozart Violin Cocerto in G major Presented by American Violinist Ray Chen</p>
+                </div>
+            </div>
+            <p class="view-program desktop-only">View whole program</p>
+
+            <div class="view-button">
+                <button type="button" class="mobile-only event-button">Download Program</button>
+            </div>
+            
+        </section>
+
+        <!-- Featured-section Section. -->
+        <section class="featured-container">
+            <h2>Featured Performers</h2>
+            <hr>
+            <div class="featured-section">
+                <div class="feature feature-1">
+                    <div class="images-section">
+                        <img class="checked-bg" src="images/person-bg.png" alt="">
+                        <img class="person-img" src="images/ph1.jpg" alt="">
+                    </div>
+                    <div class="featured-content">
+                        <p class="feature-name">Ray Chen</p>
+                        <p class="feature-title">Classical Violinist</p>
+                        <hr>
+                        <p class="feature-desc">Ray has played the violin since 3years.He brings his expertise and charm to the concert</p>
+                    </div> 
+                </div>
+                
+                <div class="feature feature-1">
+                    <div class="images-section">
+                        <img class="checked-bg" src="images/person-bg.png" alt="">
+                        <img class="person-img" src="images/ph2.jpg" alt="">
+                    </div>
+                    <div class="featured-content">
+                        <p class="feature-name">Jeniffer Cho</p>
+                        <p class="feature-title">Pianist - Melbourne</p>
+                        <hr>
+                        <p class="feature-desc">Her motto: Make the piano sing! Jennifer quite literally 
+                            does that and she will be making it sing this November for sure!</p>
+                    </div> 
+                </div>
+
+                <div class="feature feature-1 onload-display">
+                    <div class="images-section">
+                        <img class="checked-bg" src="images/person-bg.png" alt="">
+                        <img class="person-img" src="images/ph3.jpg" alt="">
+                    </div>
+                    <div class="featured-content">
+                        <p class="feature-name">William O'Haire</p>
+                        <p class="feature-title">Conductor- Italy</p>
+                        <hr>
+                        <p class="feature-desc">He inherited the Leningrad Philharmonic Orchestra in 2015  
+                            and was largely responsible for 
+                            maintaining the Austro-German tradition</p>
+                    </div> 
+                </div>
```

Comment:
- [ ] The changes to your featured section seems to have introduced alignment issues as the images are not aligning properly, also the checkered background box is not in the correct position as you can see below.
> Yours
![image](https://user-images.githubusercontent.com/90556221/170704123-e05e904c-b58a-4799-b2dc-c4b38c480768.png)


> Behance
![image](https://user-images.githubusercontent.com/90556221/170703676-770a8be0-3e24-45ff-8c4e-400c66f06734.png)

There is a tiny gap between the top of the image and the checkered box as is visible in the design, kindly align the items correctly and position the checkered box accordingly.

Reply:
done